acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -61,35 +47,20 @@
       master.install_package('puppetserver')
 
       logger.info "EC2 master found: Installing #{ENV['SHA']} build of puppet-agent."
-      # Upgrade installed puppet-agent with targeted SHA.
-      opts = {
-        :puppet_collection => 'PC1',
-        :puppet_agent_sha => ENV['SHA'],
-        :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA'] ,
-        :dev_builds_url => "http://builds.delivery.puppetlabs.net"
-      }
-
-      copy_dir_local = File.join('tmp', 'repo_configs', master['platform'])
-      release_path_end, release_file = master.puppet_agent_dev_package_info( opts[:puppet_collection], opts[:puppet_agent_version], opts)
-      release_path = "#{opts[:dev_builds_url]}/puppet-agent/#{opts[:puppet_agent_sha]}/repos/"
-      release_path << release_path_end
-      fetch_http_file(release_path, release_file, copy_dir_local)
-      scp_to master, File.join(copy_dir_local, release_file), master.external_copy_base
```

Comment:
Ohh, this might explain it... this downloads the file locally, then sends it over to the master. I may have messed this up then... Sorry @johnduarte

Reply:
https://github.com/puppetlabs/puppet/pull/6221